.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,12 @@ would find useful.
 
 We do not anymore accept cPanel like PHP+MySQL hosting services.
 
+### New Service description 
+<!-- Please fill all these details, helps us review and merge your PR quickly -->
+Service name:
+Service website:
+Service pricing page: 
```

Comment:
Problem is once it’s too long no one will read it. I think my initial one was already too wrong

Reply:
Fair. We probably need to trim it. I will try to trim it if I get some free time.